getting-started/install.md

```
@@ -0,0 +1,226 @@
+- Java is downloaded (not installed)
+- JavaFX is downloaded (not installed)
+- It creates a shortcut, which if you check it says something like: `%APPDATA%\sklauncher\jre\bin\javaw.exe -Xmx512M -jar %APPDATA%\sklauncher\SKlauncher.jar`
+
+Is it a virus?
+- Since Setup is [open source](<https://github.com/sklauncher/installer>) and compiled from the same GitHub Actions, everything the documentation says [about it](/virus) still applies in the same way.
+- Did you know that for a simple change like [this](https://github.com/sklauncher/installer/commit/9c25564cfcc51aae11965ffc50bab7a52547cae3), VirusTotal went from 6 to 2 detections? So please, let's not have this talk every time the .exe (in this case Setup) is updated.
+
+### I have Java X, but the game requires Java Y, will it work?
+Yes, it will work. SKlauncher downloads the game's version of Java, so you can use any version of Java on your computer. \*
+
+<small>\* - And if you want to use another Java when playing (like using Java 21 when the game would use Java 17), you can do it! But the explanation is [here](/faq/launcher-related#can-i-use-a-different-java-for-the-game).</small>
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)